src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -6,31 +6,45 @@ import { LogService } from '../../services/log.service';
 import { StatusMessageService } from '../../services/status-message.service';
 import { LOCAL_DATE_TIME_FORMAT, TimeResolvingResult, TimezoneService } from '../../services/timezone.service';
 import { ApiConst } from '../../types/api-const';
-import { GeneralLogEntry, GeneralLogs } from '../../types/api-output';
+import { ActionClasses, GeneralLogEntry, GeneralLogs, SourceLocation } from '../../types/api-output';
 import { LogsTableRowModel } from '../components/logs-table/logs-table-model';
 import { DateFormat } from '../components/session-edit-form/session-edit-form-model';
 import { TimeFormat } from '../components/session-edit-form/time-picker/time-picker.component';
+import { collapseAnim } from '../components/teammates-common/collapse-anim';
 import { ErrorMessageOutput } from '../error-message-output';
 
 /**
  * Model for searching of logs.
  */
 interface SearchLogsFormModel {
-  logsSeverity: Set<string>;
+  logsSeverity: string;
+  logsMinSeverity: string;
+  logsEvent: string;
+  logsFilter: string;
   logsDateFrom: DateFormat;
   logsDateTo: DateFormat;
   logsTimeFrom: TimeFormat;
   logsTimeTo: TimeFormat;
 }
 
 /**
- * Query parameters for HTTP request
+ * Query parameters for logs endpoint.
  */
-interface QueryParams {
+export interface QueryParams {
```

Comment:
If this interface is `export`ed, it becomes important to name it to a less generic name. Otherwise something may end up importing this interface, thinking it is a generic query parameter interface, only to find out otherwise.

Reply:
I will change the name of interface to a more specific one.